mappings/net/minecraft/entity/passive/AnimalEntity.mapping

```diff
@@ -16,5 +18,10 @@ CLASS ama net/minecraft/entity/passive/AnimalEntity
 	METHOD b writeCustomDataToTag (Lhs;)V
 	METHOD c canImmediatelyDespawn (D)Z
 		ARG 1 distanceSquared
+	METHOD d setLove (I)V
```

Comment:
Is this a timer? Is this a value?

Reply:
It's a boolean value that indicates that the animal is in love.

Reply:
No, it's an `int`...

Reply:
yes... looked at the wrong method